tests/test_project/tests/test_staticfinders.py

```diff
@@ -0,0 +1,9 @@
+import unittest
+
+from openwisp_utils.staticfiles import DependencyFinder
+
+
+class TestStaticFinders(unittest.TestCase):
+    def test_dependency_finder(self):
+        finder = DependencyFinder()
+        self.assertIsInstance(finder.locations, list)
```

Comment:
change to `self.assertIn('django_netjsonconfig', finder.locations[0][1]` and do something similar for `DependencyLoader`